pyproject.toml

```diff
@@ -77,6 +78,7 @@ typing = [
 ]
 
 [tool.mypy]
+disable_error_code = "operator"
```

Comment:
is this still relevant?

Reply:
I forget why it was added

Reply:
it's probably best not to have it, and to just fix whatever mypy errors you're getting